shm/src/Island.cpp

```diff
@@ -0,0 +1,10 @@
+#include "../inc/Island.hpp"
```

Comment:
No need to use relative paths. Paths are configurable in building tools (cmake). You can also configure C/C++ Tools (for Windows) with settings.json in .vscode directory to make header files visible by this extension:

{
"C_Cpp.default.includePath": [ "shm/inc" ]
}